content/blog/gatsby-tina-101.md

```diff
@@ -65,15 +65,15 @@ module.exports = {
 }
 ```
 
-I'm using Remark for all my content in _Tyra_, but TinaCMS also supports the use of JSON content using the [gatsby-tinacms-json](https://tinacms.org/docs/gatsby/json/) plugin. JSON is great for page content and [blocks](https://tinacms.org/blog/what-are-blocks/), but for simple blog posts the Remark Gatsby plugin is sufficient.
+_Tyra_ uses Markdown for content, but TinaCMS also supports JSON files via [gatsby-tinacms-json](https://tinacms.org/docs/gatsby/json/). I find that JSON is great for page content and [blocks](https://tinacms.org/blog/what-are-blocks/). But for simple blog posts, Markdown works great.
 
-Since _Tyra_ is a Git repository, all changes in content will need to be committed back into the repository. The `gatsby-tinacms-git` plugin handles the creation of new commits with content, without requiring your content authors to use Git. By default, changes are pushed to a remote branch, but the plugin can be configured with different behavior.
+Since the content _Tyra_ is Git based, all edits will need to be committed back into the repository. `gatsby-tinacms-git` tracks content changes and handles the creation of new commits with content. By default, changes are pushed to a remote branch, but the plugin is configurable.
```

Comment:
```suggestion
Since the content _Tyra_ is Git-based, all edits need to be committed back into the repository. `gatsby-tinacms-git` tracks content changes and handles the creation of new commits with content. By default, changes are pushed to a remote branch, but the plugin is configurable.
```